examples/pipeline_tacotron2/loss/tacotron2_loss_gpu_test.py

```diff
@@ -0,0 +1,41 @@
+import os
+import unittest
+
+import torch
+
+from .tacotron2_loss_impl import (
+    Tacotron2LossShapeTests,
+    Tacotron2LossTorchscriptTests,
+    Tacotron2LossGradcheckTests,
+)
+
+
+def skipIfNoCuda(test_item):
```

Comment:
~Can you add `#TODO: Find a way to de-duplicate this utility and reuse the original one from "test/torchaudio_unittest"`~

~Looks like multiple utilities are duplicated, and this soon becomes unmanageable. Can you modify `PYTHONPATH` to include the `<repo_root>/test/` directory so that you can import?~

Actually, I remembered that `example` directories are exposed in unit tests. 

https://github.com/pytorch/audio/blob/c49db7393a096ff18725603f5132707c7a3bef83/test/torchaudio_unittest/example/__init__.py#L1-L8

Can you move the test implementations into [`test/torchaudio_unittest/example`](https://github.com/pytorch/audio/tree/master/test/torchaudio_unittest/example
) directory?

Reply:
Good idea! I've moved the test implementations to  `test/torchaudio_unittest/example/tacotron2`.